core/input/keyboard.js

```diff
@@ -10,7 +10,7 @@
 
 import * as Log from '../util/logging.js';
 import { isTouchDevice } from '../util/browsers.js';
```

Comment:
Did you check these imports? `isTouchDevice` doesn't seem like something the keyboard code would need?

Reply:
I did check them but must've missed that one, good eye!